Lib/test/cov.py

```diff
@@ -0,0 +1,22 @@
+"""A minimal hook for gathering line coverage of the standard library."""
+
+import sys
```

Comment:
Please add a comment explaining that this module must not import anything but sys, and explain why.